lowatt_enedis/__init__.py

```diff
@@ -207,8 +207,13 @@ def call_service(client, args):
             try:
                 return func(client, args)
             except WebFault as exc:
-                res = exc.fault.detail.erreur.resultat
-                raise WSException("{}: {}".format(res._code, res.value))
+                try:
+                    res = exc.fault.detail.erreur.resultat
+                    raise WSException("{}: {}".format(res._code, res.value))
+                except AttributeError:
+                    # AttributeError: 'Fault' object has no attribute 'detail',
+                    # at least when server's response isn't properly parseable.
+                    raise WSException(str(exc))
```

Comment:
There's a lot of other possible `AttributeError` in the try block, I'd suggest:

```suggestion
                fault = exc.fault
                try:
                    detail = fault.detail
                except AttributeError:
                    # AttributeError: 'Fault' object has no attribute 'detail',
                    # at least when server's response isn't properly parseable.
                    raise WSException(str(exc))
                else:
                    res = detail.erreur.resultat
                    raise WSException("{}: {}".format(res._code, res.value))
```